.circleci/config.yml

```diff
@@ -88,16 +88,16 @@ jobs:
             python scripts/test.py
 
   build-docker-image:
-    executor: bionic
+    machine: true # required for testing files permissions
```

Comment:
This is kind of sad..  I it probably slower to start up and perhaps more costly.

Can you explain a little more why the regular executor doesn't work?

Reply:
I'm not sure why the regular executor didn't work. This was probably necessary to run privileged Docker containers.

I reverted the executor back to `bionic` with commit https://github.com/emscripten-core/emsdk/pull/571/commits/f5f2fbf730da002083d4c6f7f53e6908fc571dd2.

Reply:
Argh, it looks like mounting volumes doesn't work for the CircleCI Docker executor, that's why CI failed for https://github.com/emscripten-core/emsdk/commit/f5f2fbf730da002083d4c6f7f53e6908fc571dd2. See:
https://discuss.circleci.com/t/docker-compose-yml-and-volume-mounts/10043/7

Fixed with https://github.com/emscripten-core/emsdk/pull/571/commits/75ca51511f2865852109cad21cef7f846cbb2dda.